faststream/nats/broker/broker.py

```diff
@@ -901,3 +901,12 @@ async def wrapper() -> None:
                 self.__is_connected = True
 
         return wrapper
+
+    async def new_inbox(self) -> str:
+        """Return a unique inbox that can be used for NATS requests or subscriptions.
+
+        Calls `nats.aio.client.Client.new_inbox` [1] under the hood.
+
+        [1] https://nats-io.github.io/nats.py/modules.html#nats.aio.client.Client.new_inbox
+        """
+        return self._connection.new_inbox()
```

Comment:
You should add assert here to check, that the broker is already connected (and ignore this asset by noseq)

Reply:
Thanks for the hint. I only saw this comment after I added a check which raises a RuntimeError. Please let me know if you'd like me to use a regular assert statement instead.

Reply:
Please, use assert in this case. I prefer to use switchable checks for development.

Reply:
Gotcha, done. Out of curiosity, what do you mean by using "switchable checks for development"?